azurerm/resource_arm_key_vault.go

```diff
@@ -76,7 +76,7 @@ func resourceArmKeyVault() *schema.Resource {
 			"access_policy": {
 				Type:     schema.TypeList,
 				Optional: true,
-				MinItems: 1,
+				MinItems: 0,
```

Comment:
Since it is optional, you might just remove this line (`MinItems: 0`).

Reply:
Removed.